redwood-widget/src/jsMain/kotlin/app/cash/redwood/widget/RedwoodHTMLElementView.kt

```diff
@@ -67,6 +68,12 @@ private class RedwoodHTMLElementView(
       UiConfiguration(
         darkMode = colorSchemeQuery.matches,
         viewportSize = Size(width = element.offsetWidth.dp, height = element.offsetHeight.dp),
+        layoutDirection = when (element.dir) {
```

Comment:
I think this is reading the attribute as defined on this specific element, and I'm not sure it's value gets inherited from any parents.

The only thing I can find about this is using a computedStyle on the element and then reading the CSS `direction` property. But even then I don't know how to observe and react to changes in that.

We can leave this as-is for now.